internals/src/Tizen.Applications.ThemeManager/Tizen.Applications.ThemeManager/ThemeEventArgs.cs

```diff
@@ -0,0 +1,42 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Applications.ThemeManager
+{
+    /// <summary>
+    /// ThemeEventArgs class. This class is an event arguments of the ThemeLoaderChanged events.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class ThemeEventArgs : EventArgs
```

Comment:
You can use auto generated propery
``` c#
    public class ThemeEventArgs : EventArgs
    {
          internal ThemeEventArgs(Theme theme) 
          {
                 Theme = theme;
          }
          public Theme Theme { get; }
    }
```
It can resolve below warrings

Reply:
Done